### ISSUE TITLE
keyNotFound error when retrieving model with uppercase letters in column names

### ISSUE THREAD

**Issue by bagfatnick** [IBM-Swift/Swift-Kuery-ORM#65]
## Context and Description
keyNotFound error when model has uppercase letters in column names. No issue when column names are all lower case, or if retrieved via Swift-Kuery

### Environment Details
SwiftKueryORM 0.3.1
SwiftKueryPostgreSQL 1.2.0
SwiftKuery 2.0.0
Kitura 2.4.1 

Machines tested:
MacOs 10.14 Beta (18A353d), Swift version 4.2 (swiftlang-1000.0.16.9 clang-1000.10.25.3)
ibmcom/swift-ubuntu docker container v4.1.3, Swift version 4.1.3 (swift-4.1.3-RELEASE)

### Steps to Reproduce

```
struct SubjectScore: Codable, Model {
    var name: String
    var subjectCode: String
    var score: Int?
}

var subjectId = 0
    
//save a subject
let subjectScore = SubjectScore(name: "Chemistry", subjectCode: "CHEM101", score: 99)
subjectScore.save { (id: Int?, subjectScore: SubjectScore?, error: RequestError?) in
    ...
    
    subjectId = id!
    print(subjectScore)  // SubjectScore(name: "Chemistry", subjectCode: "CHEM101", score: Optional(99))
}
    
//retrieve a subject
SubjectScore.find(id: subjectId) { (subjectScore, error) in
    if let error = error {
            print(error)  // 702 : keyNotFound(CodingKeys(stringValue: "subjectCode", intValue: nil), Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "subjectCode", intValue: nil)], debugDescription: "No value for property with this key", underlyingError: nil))
            return
        }
    
    if let subjectScore = subjectScore{
        print((subjectScore) 
    }
}
```

### Expected vs. Actual Behaviour
- [ ] Expected to happen: successfully retrieved model
- [ ] Actually happened: keyNotFound error

Edit: added Kitura version

**Comment by kilnerm** [comment]
I have a duplicate issue for this #90 and a fix under review in PR #93

**Comment by kilnerm** [comment]
I have merged a fix for this issue into master. It will be available in the next release.
